### PR TITLE
Exchange Publication Plugin Integration

### PR DESCRIPTION
**Pull request by marianomerlo**

### CODE REVIEW

File: mule-artifact-it/mule-deployer-it/pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.mule.tools.maven</groupId>
         <artifactId>mule-artifact-it</artifactId>
-        <version>3.5.0-SNAPSHOT</version>
+        <version>3.5.0-SANITIES</version>
```
Comment (marianomerlo) [inline mule-artifact-it/mule-deployer-it/pom.xml:6]: Do not panic. We will rollback this before merging. We had to stop depending on SNAPSHOTS.

File: pom.xml [hunk 2]
```diff
@@ -47,7 +47,7 @@
 
         <mule.api.version>1.1.4</mule.api.version>
         <mule.core.version>4.0.0</mule.core.version>
-        <mule.maven.client.impl.version>1.6.0-SNAPSHOT</mule.maven.client.impl.version>
+        <mule.maven.client.impl.version>1.5.0</mule.maven.client.impl.version>
```
Comment (marianomerlo) [inline pom.xml:50]: Do not panic. We will rollback this before merging. We had to stop depending on SNAPSHOTS.

**Review by julianpascualMuleSoft — APPROVED** [review]

**Comment by julianpascualMuleSoft** [comment]
LGTM... Just please remember to roll back all the versions changes before merging it.

**Comment by marianomerlo** [comment]
@julianpascualMuleSoft You may want to give another review (of the last two commits), please 🙏 . 

Thank you in advance.

**Review by julianpascualMuleSoft — APPROVED** [review]

**Pull request merged**